bindata/network/egress-router/001-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
nit: this shouldn't be in `bindata/network/egress-router`, rather `bindata/egress-router`.

Reply:
why? every other network related manifest is under /network. tbh it feels like consistent to me. i.e. multus, kuryr...

Reply:
Yep, I don't see why this should not be within network?

Reply:
because things in `/bindata/network/` are rendered by the network (operconfig) controller. This is a separate controller, and thus should have its own folder.

(If there's anything in `/bindata/network` that isn't rendered by operconfig, it should ultimately be moved)